src/app/static/src/tests/components/downloadResults/uploadModal.test.ts

```diff
@@ -204,52 +210,60 @@ describe(`uploadModal `, () => {
 
         const store = wrapper.vm.$store
         store.state.language = Language.fr;
-        expect(mockTooltip.mock.calls[4][1].value).toBe("Créer une version étiquetée dans l'ensemble de données ADR, pour inclure tous les fichiers d'entrée et de sortie");
-        expect(mockTooltip.mock.calls[5][1].value).toBe("Télécharger de nouvelles versions des fichiers sélectionnés sans créer de version");
+        await nextTick();
+        expect(mockTooltip.mock.calls[2][1].value).toBe("Créer une version étiquetée dans l'ensemble de données ADR, pour inclure tous les fichiers d'entrée et de sortie");
+        expect(mockTooltip.mock.calls[3][1].value).toBe("Télécharger de nouvelles versions des fichiers sélectionnés sans créer de version");
```

Comment:
I actually have no idea! But I feel like 0,1,2,3,4,5 is better than 0,1,4,5,6,7 anyway haha, my best guess would be in the past since vue seemed to be doing a lot of lifecycles without the `await nextTick()`s it might've done a lifecycle when we try and get the `store` out of the wrapper, which would explain the 2 & 3 going missing in the original sequence

Reply:
Ah yep that makes sense